api/db/repositories/customerRepository.js

```diff
@@ -34,4 +34,32 @@ exports.deleteCustomer = async (customerId) => {
     } catch (error) {
         throw new Error(error);
     }
+};
+
+exports.getAll = async (customers) => {
```

Comment:
`getAll` shouldn't have `customers` as an argument, in fact it doesn't require any kind of argument